torch/fx/node.py

```diff
@@ -64,22 +78,23 @@ def replace_all_uses_with(self, replace_with : 'Node') -> List['Node']:
         Replace all uses of `self` in the Graph with the Node `replace_with`.
         Returns the list of nodes on which this change was made.
         """
-        use_nodes : List[Node] = self.find_uses()
-        for use_node in use_nodes:
+        to_process = list(self.users)
+        for use_node in to_process:
             def maybe_replace_node(n : Node) -> Node:
                 if n == self:
-                    self.uses -= 1
                     return replace_with
                 else:
                     return n
+
```

Comment:
by assigning args/kwargs separately here there are two passes that have to fix up the uses might want to directly call `_update_kwargs_args`